api/custom-map-ui/geoname-search-ui.html

```diff
@@ -117,7 +117,7 @@
         <button onclick="updateSearchResults()" class="search_button btn btn-primary btn-sm" id="searchBtn">Search</button>
         
         <br>
-        <output id="results_count" for="query"></output>
+        <output role="description" id="results_count" for="query"></output>
```

Comment:
Is there a need to change [the implicit `role="status"` of the output element](https://www.w3.org/TR/html-aria/#el-output)? Also, `description` is not a valid role, you may find https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles useful.